src/fprime_gds/executables/run_deployment.py

```diff
@@ -148,6 +154,54 @@ def launch_comm(parsed_args):
     app_cmd = BASE_MODULE_ARGUMENTS + ["fprime_gds.executables.comm"] + arguments
     return launch_process(app_cmd, name=f'comm[{parsed_args.adapter}] Application', launch_time=1)
 
+def get_openmct_json(parsed_args):
+    """ Convert F-Prime Topology App Dictionary XML to Python Dictionary
+    
+    Args:
+        parsed_args: parsed argument namespace
+    Return:
+        converted OpenMCT JSON Telemetry Definitions
+        Initial States JSON for OpenMCT 
+    """
+
+    openmct_dir = fprime_openmct.__file__.replace('__init__.py', 'src/fprime_openmct/javascript')
```

Comment:
I'm not sure I understand why the `openmct_dir` is needed only to be passed as an argument in the openmct functions down below. It may be better to internalize this directly into the fprime-openmct package.

Reply:
you are right, that is not needed. I have internalized this in my latest commit